pkg/client/outofband/client.go

```diff
@@ -29,15 +27,13 @@ const (
 // RequestOptions allow you to customize the way request messages are built.
 type RequestOptions func(*Request) error
 
-// ConnectionRecorder records connection records produced as byproducts of creation of messages.
-type connectionRecorder interface {
-	SaveInvitation(id string, i interface{}) error
+type oobService interface {
+	AcceptRequest(request *outofband.Request) (string, error)
+	SaveRequest(request *outofband.Request) error
```

Comment:
Just to double-check, have we been using the `Save` terminology elsewhere?

Reply:
`connection.Recorder` has `Save*` methods.

There is `saveCredential` in the JS API and `SaveCredential` in the REST interface.

There are numerous other unexported `save*` methods elsewhere.